Penteract/Assets/Scenes/Level2.scene

```diff
@@ -240,11 +240,6 @@
                                 "Type": "GameObjectUID",
                                 "Value": 3685415630762953669
                             },
-                            {
-                                "Name": "playerUID",
-                                "Type": "GameObjectUID",
-                                "Value": 15261592491771467264
-                            },
                             {
                                 "Name": "gameCameraUID",
```

Comment:
Why was has level2 been modified?

Does it include anything important?

Reply:
dunno it wasn't me

Reply:
Level 2 has been modified in #301 too. Care when merging with that, maybe communicate with Telm